src/garage/sampler/ray_batched_sampler.py

```diff
@@ -0,0 +1,216 @@
+"""This is an implementation of an on policy batch sampler.
```

Comment:
Please rename this file to `ray_sampler.py` (or rename the class to `RayBatchSampler`) to keep the file and class names consistent.

Reply:
fixed